workflow/rocoto/gfs_tasks.py

```diff
@@ -156,7 +156,7 @@ def aerosol_init(self):
             interval = self._base['INTERVAL_GFS']
         elif self.cdump in ['gdas']:
             interval = self._base['INTERVAL']
-        offset = f'-{timedelta_to_HMS(interval)}'
+        offset = timedelta_to_HMS(-interval)
```

Comment:
so if L1013 works, why does this not?

Reply:
This worked, but I changed this to be consistent with L502 (it is the cleaner way to do it).

Reply:
ok. thanks.  I was trying to understand what was the "bug" part of this PR.

Reply:
Bug was the one on L502 that wasn't using `timedelta_to_HMS()`.